### DISCUSSION TITLE
创建服务 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#844] [Announcements]
# 创建服务 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kpanda/07UserGuide/ServicesandRoutes/CreatingServices/

**Comment by liyuerich** [comment]
在左侧导航栏中，点击服务与路由进入服务列表，点击右上角创建服务按钮。

截图和文字已经和目前产品版本不一致

**Reply by Michelle951** [reply]
> 在左侧导航栏中，点击服务与路由进入服务列表，点击右上角创建服务按钮。
> 
> 截图和文字已经和目前产品版本不一致

Fixed. Thanks!

**Comment by liyuerich** [comment]
网络配置 的说明在那里能找到？

**Reply by Michelle951** [reply]
> 网络配置 的说明在那里能找到？

@Jeanine-tw

**Reply by Jeanine-tw** [reply]
本周会完善
